src/panels/config/integrations/ha-config-integrations.ts

```diff
@@ -475,8 +475,10 @@ class HaConfigIntegrations extends SubscribeMixin(LitElement) {
         </div>
         <mwc-fab
           slot="fab"
+          extended
           aria-label=${this.hass.localize("ui.panel.config.integrations.new")}
           title=${this.hass.localize("ui.panel.config.integrations.new")}
+          label=${this.hass.localize("ui.panel.config.integrations.new_label")}
```

Comment:
```suggestion
          .label=${this.hass.localize("ui.panel.config.integrations.new_label")}
```

Reply:
Sorry I disregarded this previously. TIL about Lit's properties vs attributes… a distinction React doesn't appear to have!